packages/material-ui-styles/src/makeStyles.test.js

```diff
@@ -0,0 +1,50 @@
+import { assert } from 'chai';
+import React from 'react';
+import { createMount } from '@material-ui/core/test-utils';
+import makeStyles from './makeStyles';
+
+describe('makeStyles', () => {
+  let mount;
+
+  /**
+   * returns a function that given the props for the styles object will return
+   * the css classes
+   * @param {object} styles argument for `makeStyles`
+   */
+  function createGetClasses(styles) {
+    const useStyles = makeStyles(styles);
+    let classes = {};
+
+    function TestComponent(props) {
+      classes = useStyles(props);
+      return null;
+    }
+
+    return function getClasses(props) {
+      mount(<TestComponent {...props} />);
```

Comment:
`enzyme` does not support hooks in `shallow` rendering yet: facebook/react#14091